### PR TITLE
Add fram->frame and friend

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -22445,6 +22445,7 @@ fragmeneting->fragmenting
 fragmenets->fragments
 fragmentization->fragmentation
 fragmnet->fragment
+fram->frame
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:22448]: Other suggestions: `gram`, `fam`, `dram`, `cram`.

I don't know if there are actually such typos out there.

Comment (peternewman) [inline codespell_lib/data/dictionary.txt:22448]:
```suggestion
fram->frame, gram, dram, cram,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -22458,6 +22459,7 @@ frameword->framework
 frameworkk->framework
 framlayout->framelayout
 framming->framing
+frams->frames
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:22462]:
```suggestion
frams->frames, grams, drams, crams,
```

**Review by DimitriPapadopoulos — APPROVED** [review]

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt:22448]: Do you mean fam as in the informal for family?

Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:22448]: Yes. I guess that would be a very rare typo. Not sure it is worth it.

Comment (peternewman) [reply codespell_lib/data/dictionary.txt:22448]: I also wasn't sure if aspell would be happy with it...

**Pull request merged**